### ISSUE TITLE
programs that are located on mounted/removable drives -still- don't get recognized after unmount/mount

### ISSUE THREAD

**Issue by tokariu** [henrypp/simplewall#210]
I'm sorry to reopen this as a new issue, but you've marked the other as fixed, but it's actually not fixed (at least it's still not working for me, and maybe others).

the issue: https://github.com/henrypp/simplewall/issues/128 is still applicable. 

https://github.com/henrypp/simplewall/issues/128#issuecomment-388785691 is still reproducable.

**Comment by ltGuillaume** [comment]
I can confirm this behavior. Only after restarting simplewall, I can get a popup concerning an application on a Pismo Private Folder (which is a mounted virtual volume).
